ci/ray_ci/test_ray_docker_container.py

```diff
@@ -24,16 +24,16 @@ def _mock_run_script(input: List[str]) -> None:
             "ci.ray_ci.docker_container.LinuxContainer.run_script",
             side_effect=_mock_run_script,
         ):
-            container = RayDockerContainer("3.8", "cu11.8.0", "ray")
+            container = RayDockerContainer("3.9", "cu11.8.0", "ray")
```

Comment:
maybe just changes these to a fake version number? like "3.99"? so that we won't need to update these tests in the future?